k8-util/helm/fluvio-sys/templates/crd_topic.yaml

```diff
@@ -90,6 +90,33 @@ spec:
                     maxPartitionSize:
                       type: integer
                       minimum: 2048
+                deduplication:
```

Comment:
Bit concerned about duplicating schema.   let's create issue to see if we can have better reusability